sdk/include/xdk/ntbasedef.h

```diff
@@ -788,7 +788,11 @@ extern "C++" { \
 #endif /* _M_AMD64 */
 
 /* C_ASSERT Definition */
-#define C_ASSERT(expr) extern char (*c_assert(void)) [(expr) ? 1 : -1]
+#ifndef _Static_assert
+ #define C_ASSERT(expr) typedef char c_assert [(expr) ? 1 : -1]
+#else
+ #define C_ASSERT(expr) _Static_assert(expr, "(" #expr ") failed")
+#endif
```

Comment:
Why is this a problem with Clang? Does it fail to compile even if the static assertion is true?

Reply:
As far as I understand, it just doesn't like how we define our `C_ASSERT`

Reply:
The way we do it is the same as the MS PSDK or something like that 🤔